PopulateTools/gobierto
# 918 - Polish process creation UX

Connects to #918 

### What does this PR do?

Implements a series of changes so creating a new process/group is less restrictive and more user friendly (see details below).

**IMPORTANT:** from now on, when a process is created all of its stages records are created, but with the flat `active: true`. This PR contains a data migration to update all the previously created process stages.

### How should this be manually tested?

When creating a process from the admin, the following things should have changed:

* When you first create a process / group, the part of the form dedicated to create stages is hidden.
* After creating the process, you should be able to add new stages.
* As long as a stage is not activated, validations shouldn't avoid it from being saved.
* Starts / ends is no longer compulsory for processes. There is a checkbox to toggle this.

### Does this PR changes any configuration file?

Nope
